SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReplicationGraphLoadBalancingHandler.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialReplicationGraphLoadBalancingHandler.h"
+#include "EngineClasses/SpatialReplicationGraph.h"
```

Comment:
Should separate the header include for this .cpp file into its own group otherwise afaik clang-format will sort it below
```suggestion

#include "EngineClasses/SpatialReplicationGraph.h"
```